### ISSUE TITLE
About the weight bias

### ISSUE THREAD

**Issue by XDUSPONGE** [fangwei123456/spikingjelly#174]
Hello, thanks for the wonderful work.  I have some questions about the weight set about the paper "Incorporating Learnable Membrane Time Constant to Enhance Learning of Spiking Neural Networks".
* Why the weight bias is set with false?
* The version in the spikingjelly is different with the code the paper given, in which the conv bias is false and fc bias is true, and another one conv and fc bias both set false, which version can reproduce the result in the paper, thanks

**Comment by fangwei123456** [comment]
> Why the weight bias is set with false?

Because BN has bias, and then the bias of Conv is redundant.

> The version in the spikingjelly is different with the code the paper given, in which the conv bias is false and fc bias is true, and another one conv and fc bias both set false, which version can reproduce the result in the paper, thanks

Both of them can reproduce the nearly same results. The bias of FC has very limited influence on accuracy. The identical results (e.g., the same tensorboard logs) can not be guaranteed (see https://pytorch.org/docs/stable/notes/randomness.html).

**Comment by XDUSPONGE** [comment]
> > Why the weight bias is set with false?
> 
> Because BN has bias, and then the bias of Conv is redundant.
> 
> > The version in the spikingjelly is different with the code the paper given, in which the conv bias is false and fc bias is true, and another one conv and fc bias both set false, which version can reproduce the result in the paper, thanks
> 
> Both of them can reproduce the nearly same results. The bias of FC has very limited influence on accuracy. The identical results (e.g., the same tensorboard logs) can not be guaranteed (see https://pytorch.org/docs/stable/notes/randomness.html).

Thanks for ur reply
